### DISCUSSION TITLE
Unable to send the proof of my student affiliation with a minimal acceptable quality in 1mb.

### DISCUSSION THREAD

**Discussion by JoseGonz11** [community/community#139251] [GitHub Education]
### Hi everyone,

I am enrolled in a new official distance only course. The only way to fill the documentation to enter is online, and I will only recive PDFs. I cannot do a photo of my screen as is unreadable. The only proof I can send is fully digital, even though, signed from official sources, but on a image is impossible to capture neither any digital text nor any signature the file has. So at the end, seems like I have no chance to send any proof, even I got it. Te resolution I'm talking about is 542x406px.

Also I am concerned about the new course type, even if it is distance only, wich now the only way is picking the subjects you want/can to do, wich is new here, it's just the same, but distance and you do the subjects as you can or need, being able to leave some behind for a later year. May be too new to be accepted maybe?

**Comment by JoseGonz11** [comment]
By the way, I have no issue sending official documents, neither doing whatever it takes to be verified.

I am even using OBSStudio Virtual Camera to send full quality images, but no result.

**Comment by davevad93** [comment]
Hi @JoseGonz11 ,

If possible, since you are a distance learner, I suggest you try to take a screenshot of some kind of academic documentation showing your name and a current date, e.g. when I applied last year, I uploaded a screenshot of my last exam result and was accepted instantly.

For more information, see also the [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352)

**Reply by JoseGonz11** [reply]
Sure, I already uploaded screenshots, from everything, even my apply, the gubernamental response to that, even the portal that says I paid, how much and when. This page styling with so much white space, ain't no way a 400x500 image could read any of this, even zooming in as the image is a square. This is from the official andalusian portal. It says I entered course on 2024, and wich subjects I am doing.

![image](https://github.com/user-attachments/assets/1903c60e-8e8d-48e1-83c8-1a92f3d22827)

**Reply by davevad93** [reply]
Hey @JoseGonz11 , I see, however looking at the screenshot I don't see a current date, i.e. the most recent one is `31/07/2024` which is 2 months ago, I say this because initially when I've submitted my application , I uploaded a photo of my enrollment sheet dated a couple of months before and it was rejected for that reason alone. But the screenshot of my last exam showed a current date and that's why I was accepted. They need a current or very close date in a `DD/MM/YYYY` or `MM/DD/YYYY` format so they can verify your current student status.

**Reply by JoseGonz11** [reply]
Okay, anyway I have requested a document proving I am studiyng to the center I'm in, hope that works. Thanks for the tip.

**Reply by davevad93** [reply]
You're welcome man, hope you get it sorted out soon. Buen día! 👍

**Comment by AESMatias** [comment]
Hello Jose, i've been in the same situation before, i recommend you to use some online PDF compressor in order to reduce the size of the PDF file, or just making a new PDF by grouping a bunch of screenshots, but i'm not sure if this would work, the recommend approach should be request a PDF file signed by the institution, i would try to compress the size somehow.

Please verify that the current date of your student status appears clearly in the document.
Good luck!

**Comment by JoseGonz11** [comment]
I've acquired an official document telling the details, so this info is no longer missing. But I am a distance student and it says I am not near the school (I am ~250 kilometers away) even though I ticked the distance coursework

**Reply by davevad93** [reply]
I see @JoseGonz11 , well I don't know why it happens given that in the "[You appear to not be near any campus location for the school you have selected.](https://github.com/orgs/community/discussions/111352#not-near-any-campus-location-for-selected-school)" section of the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352#not-near-any-campus-location-for-selected-school) it says the following:

- ***If you are a distance learning student, please attach official, dated proof indicating that your program is a distance learning program or you are a distance learning student when you reapply.***

And from what you are saying that's what you just did, so I suggest you do one thing, contact support by opening a ticket [here](https://support.github.com/contact/education), select "`My selected school has incorrect or incomplete information e.g. domains or campus location`" and explain your issue in detail by attaching also the screenshots of your academic documentation.
